operation/fhir-operation-bulkdata/src/test/java/com/ibm/fhir/operation/bulkdata/util/BulkDataImportUtilTest.java

```diff
@@ -155,6 +158,15 @@ public void testCheckAllowedTotalSizeForTenantOrSystemFail() throws FHIROperatio
         util.checkAllowedTotalSizeForTenantOrSystem(100);
     }
 
+    @Test//(expectedExceptions = { FHIROperationException.class })
```

Comment:
why is expectedExceptions commented out?
can it be removed?

Reply:
Fixed, it should have been expected